vllm/model_executor/models/mixtral.py

```diff
@@ -189,6 +208,10 @@ def process_weights_after_loading(self):
                                           dtype=torch.float8_e4m3fn)
             w2_weight = torch.empty_like(self.w2_weight.data,
                                          dtype=torch.float8_e4m3fn)
+
+            self.w13_scale = nn.Parameter(torch.zeros(self.num_total_experts,
```

Comment:
Why reset this member again? It looks like it was made earlier.

Reply:
Because now I initialed it to (experts, 2). This is indeed not a clean way so I'm looking forward to see the RFC of weight loading refactoring 😀